src/main/proto/ga4gh/rna_quantification.proto

```diff
@@ -0,0 +1,94 @@
+syntax = "proto3";
+
+package ga4gh;
+
+import "google/protobuf/struct.proto";
+
+// Units for expression level.
+// FPKM - number of Fragments Per Kilobase of feature length per Million reads
+// FPKM is calculated by dividing the fragment count per feature by the total
+// number of reads in millions (FPM - Fragments Per Million).  FPM is then
+// divided by feature length in kilobases to obtain FPKM.
+
+// TPM - Transcripts per kilobase Per Million reads
+// TPM is calculated by first dividing the fragment/read count by feature length
+// in kilobases (RPK - Reads Per Kilobase).  The count of all RPKs in the sample
+// are then divided by a million to generate a 'per million' scaling value.  For
+// each feature RPK divided by the 'per million' scaling factor generated TPM.
+
+enum ExpressionUnit {
+  EXPRESSION_UNIT_UNSPECIFIED = 0;
+  FPKM = 1;
+  TPM = 2;
+}
+
+// Top level identifying information
+message RnaQuantification {
+  // The unique ID assigned to the results of running the described programs
+  // on the specified reads and assignment to the listed annotation.
+  string id = 1;
+ 
+  // Name
+  string name = 2;
```

Comment:
Is this `name` used to associate with a given sample? I assume, since there's a `read_group_id` that that's the mechanism for associating these with other data in the server...